src/main/java/teammates/common/datatransfer/StudentProfileAttributes.java

```diff
@@ -65,90 +66,97 @@ public StudentProfileAttributes() {
         this.pictureKey = "";
         this.modifiedDate = null;
     }
-    
+
     public String generateUpdateMessageForStudent() {
-        String reminder = "";
-        
         if (isMultipleFieldsEmpty()) {
-            reminder = Const.StatusMessages.STUDENT_UPDATE_PROFILE; 
+            return Const.StatusMessages.STUDENT_UPDATE_PROFILE;
         } else {
             if (this.shortName.isEmpty()) {
-                reminder = Const.StatusMessages.STUDENT_UPDATE_PROFILE_SHORTNAME; 
+                return Const.StatusMessages.STUDENT_UPDATE_PROFILE_SHORTNAME;
             } else if (this.email.isEmpty()) {
-                reminder = Const.StatusMessages.STUDENT_UPDATE_PROFILE_EMAIL; 
+                return Const.StatusMessages.STUDENT_UPDATE_PROFILE_EMAIL;
             } else if (this.pictureKey.isEmpty()) {
-                reminder = Const.StatusMessages.STUDENT_UPDATE_PROFILE_PICTURE;
+                return Const.StatusMessages.STUDENT_UPDATE_PROFILE_PICTURE;
             } else if (this.moreInfo.isEmpty()) {
-                reminder = Const.StatusMessages.STUDENT_UPDATE_PROFILE_MOREINFO;
+                return Const.StatusMessages.STUDENT_UPDATE_PROFILE_MOREINFO;
             } else if (this.nationality.isEmpty()) {
-                reminder = Const.StatusMessages.STUDENT_UPDATE_PROFILE_NATIONALITY;
+                return Const.StatusMessages.STUDENT_UPDATE_PROFILE_NATIONALITY;
             }
+            return "";
         }
-        
-        return reminder;
     }
-    
+
     private boolean isMultipleFieldsEmpty() {
-        int numEmptyFields = 0;
-        if (this.shortName.isEmpty()) {
-            numEmptyFields ++;
-        }
-        if (this.email.isEmpty()) {
-            numEmptyFields ++;
-        }
-        if (this.nationality.isEmpty()) {
-            numEmptyFields ++;
-        }
-        if (this.moreInfo.isEmpty()) {
-            numEmptyFields ++;
-        }
-        if (this.pictureKey.isEmpty()) {
-            numEmptyFields ++;
-        }
-        
+        int numEmptyFields = addCounterIfEmpty(shortName)
+                + addCounterIfEmpty(email)
+                + addCounterIfEmpty(nationality)
+                + addCounterIfEmpty(moreInfo)
+                + addCounterIfEmpty(pictureKey);
         return numEmptyFields > 1;
     }
 
+    private int addCounterIfEmpty(String str) {
```

Comment:
This method is not involve any 'adding' (or a 'counter'), so the method name is not appropriate. Name methods based on what it does, not how it is used.